init_database.py

```
@@ -0,0 +1,172 @@
+
+            print(f"  {len(song_ids)} 曲見つかりました")
+
+            # MySQLに移行
+            migrated = 0
+            skipped = 0
+
+            with get_session() as session:
+                for i, song_id in enumerate(song_ids):
+                    metadata = metadatas[i] if i < len(metadatas) else {}
+
+                    # メタデータがない場合はスキップ（新形式のデータ）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)